GOVERNANCE.md

```diff
@@ -0,0 +1,201 @@
+# ESLint-Community Governance
+
+## Core Team
+
+The Core Team are the curators and stewards of the ESLint-Community project . Their key
+responsibility is to evaluate new projects that are proposed to join `@eslint-community`
+and to help out in offering a base level maintenance for all primary projects within
+the organization.
+
+The Core Team should have occasional meetings to discuss the project, with meeting minutes
+being added to [eslint-community/governance][].
+
+The Core Team are together with the ESLint TSC the organization owners and the only ones
+with `admin` access to repositories within the organization.
+
+The Core Team are the only members of the `@eslint-community/core-team` team.
+
+The Core Team and ESLint TSC have a shared `#community-team` chat room, in the ESLint
+Discord, to communicate with one another.
```

Comment:
What about linking to the Discord server as well?

Reply:
If there's a good link, then sure 👍

Reply:
We used https://eslint.org/chat/eslint-community in the org announcement post

Reply:
Done ✔️